aptos-move/framework/src/built_package.rs

```diff
@@ -107,6 +110,8 @@ pub struct BuildOptions {
     /// Select bytecode, language, compiler for Move 2
     #[clap(long)]
     pub move_2: bool,
+    #[clap(long, default_value = "human")]
```

Comment:
Instead of another field, could we just reuse `experiments` above, and have this be another "experiment" (that is, if the experiment is on, then we have JSON formatted messages, e.g.)? See `third_party/move/move-compiler-v2/src/experiments.rs` and the function `experiments_from_opt_level` for an example of threading through an "option" via experiments.

Reply:
Yes, I think it's a better option. I'll rewrite the PR now, removing the `message_format` field, thanks!